content/applications/finance/fiscal_localizations/peru.rst

```diff
@@ -731,7 +731,15 @@ Next, you need to retrieve the *client ID* and *client secret* from |SUNAT|. To
    differ from the user set for electronic invoicing.
 
 These credentials should be used to configure the delivery guide general settings from
-:menuselection:`Accounting --> Configuration --> Settings --> Peruvian Electronic Invoicing`.
+:menuselection:`Inventory --> Configuration --> Settings`, and scroll down to the :guilabel:`Peru
+Delivery Guide` section.
+
+Configure the following :guilabel:`Sunat Delivery Guide API` fields:
+
+- :guilabel:`Guide Client ID`
+- :guilabel:`Guide Client Secret`
```

Comment:
Wouldn't it be helpful for the user to add more information explaining the Guide Client ID and Guide Client Secret?

Reply:
Excellent improvement, thanks guys @afma-odoo @samueljlieber